### PR TITLE
[nrf fromlist] soc: nordic: RRAMC regions

### PR DESCRIPTION
**Pull request by michalek-no**
Adding definition of regions granularity to Kconfig.

Upstream PR: https://github.com/zephyrproject-rtos/zephyr/pull/72509

### CODE REVIEW

File: soc/nordic/Kconfig [hunk 1]
```diff
@@ -73,6 +73,13 @@ config NRF_ACL_FLASH_REGION_SIZE
 	help
 	  FLASH region size for the NRF_ACL peripheral.
 
+config NRF_RRAM_REGION_SIZE_GRANULARITY
```
Comment (hubertmis) [inline soc/nordic/Kconfig:76]: This value applies to the RRAM peripheral, not to all Nordic SoCs. I think it should be placed in https://github.com/nrfconnect/sdk-zephyr/blob/main/drivers/flash/Kconfig.nrf_rram instead.

**Review by hubertmis — APPROVED** [review]

**Review by nvlsianpu — APPROVED** [review]

**Review by de-nordic — APPROVED** [review]

**Review by sigvartmh — APPROVED** [review]

**Review by anangl — APPROVED** [review]

**Review by hubertmis — APPROVED** [review]

**Comment by michalek-no** [comment]
rebase

**Review by nvlsianpu — APPROVED** [review]

**Comment by michalek-no** [comment]
rebase

**Comment by nvlsianpu** [comment]
Already in https://github.com/nrfconnect/sdk-zephyr/commit/a85919141368b9a11b995f47d18bacfe209ed3e3

**Pull request closed without merging**